core/src/components/modal/gestures/sheet.ts

```diff
@@ -80,6 +102,15 @@ export const createSheetGesture = (
       return false;
     }
 
+    /* tslint:disable-next-line */
+    if (typeof document !== 'undefined') {
+      const activeElement = baseEl.ownerDocument.activeElement as HTMLElement;
+      if (activeElement.matches('input,ion-input,textarea,ion-textarea')) {
+        // Dismisses the open keyboard when the sheet drag gesture is started.
+        activeElement.blur();
+      }
+    }
+
```

Comment:
Putting this in `canStart` feels a little weird; it seems like it should go in `onStart` or somewhere else that more explicitly indicates that we're starting a gesture.

Reply:
Agreed 🤔  let me confirm that doing this behavior while the gesture is starting, doesn't cause any other issues. Then I'll move it 👍